pytorch-yolo-image/README.md

```diff
@@ -0,0 +1,32 @@
+# YOLOV8 detection Example For WASI-NN with PyTorch Backend
+
+Code for a working example of object detection using the yolov8 model. 
+
+To build (from the `/rust` directory):  
+`cargo build --release --target wasm32-wasi`
+
+To compile the WASM AOT (Ahead of Time) - this results in a much more performant binary  
+```bash
+wasmedge compile ./target/wasm32-wasi/release/wasmedge-wasinn-example-yolo-image.wasm ./target/wasm32-wasi/release/wasmedge-wasinn-example-yolo-image.wasm
+```
+
```

Comment:
Instead of a note to install yolov8, perhaps the instructions can be directly added here on how to install Yolov8 

Also it should include instruction to install torchvision: `pip3 install torchvision`

Such instructions have been added in the Pytorch example for Mobilenet v2. 

`python3 get_model.py`